src/Controller/EnMarche/SummaryManagerController.php

```diff
@@ -86,4 +89,49 @@ public function removeExperienceAction(Request $request, JobExperience $experien
 
         return $this->redirectToRoute('app_summary_manager_index');
     }
+
+    /**
+     * @Route("/skills", name="app_summary_manager_handle_skills")
+     * @Method("GET|POST")
+     */
+    public function handleSkillAction(Request $request)
+    {
+        $this->disableInProduction();
+
+        $summaryManager = $this->get(SummaryManager::class);
+        $summary = $summaryManager->getForAdherent($this->getUser());
+        $form = $this->createForm(SummaryType::class, $summary, [
+            'step' => SummaryType::STEP_SKILLS,
+            'action' => $this->generateUrl('app_summary_manager_handle_skills'),
+        ]);
+
+        if ($form->handleRequest($request)->isSubmitted() && $form->isValid()) {
+            $this->addFlash('info', $this->get('translator')->trans('summary.handle_skills.success'));
+
+            return $this->redirectToRoute('app_summary_manager_index');
+        }
+
+        return $this->render('summary_manager/handle_skills.html.twig', [
+            'form' => $form->createView(),
+        ]);
+    }
+
+    /**
+     * @Route("/skill_autocomplete", name="app_summary_manager_skill_autocomplete")
+     */
+    public function skillAutocompleteAction(Request $request)
+    {
+        $this->disableInProduction();
+
+        $names = array();
```

Comment:
You can leverage PHP7 syntax to remove this line and return instead, btw we use short array syntax:
```php
return new JsonResponse($names ?? []);
```